src/AgGrid.elm

```diff
@@ -304,6 +304,8 @@ type alias ColumnDef dataType =
 -}
 type alias ColumnSettings =
     { aggFunc : Aggregation
+    , allowedAggFuncs : List String
+    , defaultAggFunc : String
```

Comment:
We should use the existing `Aggregation` type for both to prevent wrong configurations

Reply:
No idea why I didn't do that in the first place :shrug: 

Note that now our API allows to set `null` for `defaultAggFunc` (because `AgGrid.NoAggregation` will serialize to that). The AgGrid API does not explicitly allow that. So I tested it and AgGrid does not crash and properly reports `ag-grid-enterprise.auto.esm.js:3419 AG Grid: unrecognised aggregation function null` in the console. So I think this is OK.

Reply:
I don't even see an error myself when trying this..

Nonetheless, I'm not sure if we want to let it fail in this case. It's not very Elm'ish being able to set a value that leads to a runtime error. We could just prevent this when encoding the values and filter it.

Reply:
Done (no idea why I thought we needed that)